app/models/user.rb

```diff
@@ -38,4 +40,8 @@ class User < ApplicationRecord
   # :lockable, :timeoutable, :trackable and :omniauthable
   devise :database_authenticatable, :registerable, :confirmable,
          :recoverable, :rememberable, :validatable
+
+  def company?
```

Comment:
I see no need in this method. It's not used anywhere. Also such things should be in delegator